article/app/views/fragments/articleAsideSlot.scala.html

```diff
@@ -0,0 +1,17 @@
+@(
+    shouldShowAds: Boolean,
+    optionalSlotSizes: Seq[String],
+    optClassNames: Option[String] = None
+
+)
+@import conf.switches.Switches.CommercialSwitch
+
+@if(CommercialSwitch.isSwitchedOn && shouldShowAds) {
+  @fragments.commercial.adSlot(
```

Comment:
Each template that includes the fragment currently wraps it in a `<div class="ad-slot-container">`

I wanted to keep the template consistent with the gallerySlot and sliceSlot; happy to move the ad wrapper div to the template too, if it would be cleaner?